content/docs/command-reference/plots/index.md

```diff
@@ -96,7 +87,44 @@ names in the `train` array below:
 }
 ```
 
-## Plot templates (data series only)
+## Definining a plot
+
+In order to create visualizations, users need to provide the data and
+(optionally) configuration that will help customize the plot. DVC provides two
+ways to configure visualizations. Users can mark specific <abbr>stage</abbr>
+<abbr>outputs</abbr> as plot or define plot configuration inside `dvc.yaml`
+under `plots` key.
+
+## Plots definitions
+
+Plots defined in `dvc.yaml` are especially useful when users want to compare
+data from differend data sources residing on the same version of the project.
+For example, comparing training versus test results on current branch.
+
+### Syntax
+
+In order to define the plot users need to provide data and configuration for the
+plot. The plots should be defined in `dvc.yaml` file under `plots` key. Refer to
+the [examples](/doc/command-reference/plots#example-simple-plot-definition) for
+more syntax insight.
+
+```yaml
+# dvc.yaml
+stages: ...
+
+plots: ...
+```
+
+## Plot outputs
```

Comment:
By reading this I don't understand how plot outputs are not related to the global "plots" section. Could we clarify this?

Reply:
See also https://github.com/iterative/example-repos-dev/pull/117#issuecomment-1175197428